webpack.config.js

```diff
@@ -59,11 +59,49 @@ module.exports = function (config, { isClient }) {
             name: 'sfui',
             priority: 2
           },
+          filepond: {
+            test: /filepond/,
+            name: 'filepond',
+            priority: 2
+          },
+          slickCarousel: {
+            test: /vue-slick-carousel/,
+            name: 'vue-slick-carousel',
+            priority: 2
+          },
+          vue2Datepicker: {
+            test: /vue2-datepicker/,
+            name: 'vue2-datepicker',
+            priority: 2
+          },
+          jquery: {
+            test: /jquery/,
+            name: 'jquery',
+            priority: 2
+          },
+          vueMultiselect: {
+            test: /vue-multiselect/,
+            name: 'vue-multiselect',
+            priority: 2
+          },
+          croppie: {
+            test: /croppie/,
+            name: 'croppie',
+            priority: 2
+          },
+          braintreeWeb: {
+            test: /braintree-web/,
+            name: 'braintree-web',
+            chunks: 'async',
+            enforce: true,
+            priority: 2
+          },
           vendorInitial: {
             // create 'vendor' group from initial packages from node_modules except Storefront UI
             test: /node_modules/,
             name: 'vendor-initial',
```

Comment:
what is included into `vendor-initial` and `vendor-async` chunks?
Is there easy way to see it?

Reply:
@gorbunovav enable `BundleAnalyzerPlugin` in `webpack.client.config.ts`. It will generate `report.html` in dist directory. `vendor-async` became pretty small, but it's includes some libs that used in `storyblok` and `storefront-ui` components. So it's loaded for each pages. I'll describe this in issue and probably we can get rid of `vendor-async` chunk or at least it will not load on each page.